kernel/configs/fragments/common/virtio.conf

```diff
@@ -0,0 +1,25 @@
+# We need virtio for 9p and serial and vsock at least
+
+# To get VIRTIO, we need a bus - ours of choice is PCI. We need to enable
+# PCI support to get VIRTIO_PCI support
+CONFIG_PCI=y
+CONFIG_PCI_MSI=y
+CONFIG_PCI_MSI_IRQ_DOMAIN=y
+# To get to the VIRTIO_PCI, we need the VIRTIO_MENU enabled
+CONFIG_VIRTIO_MENU=y
+CONFIG_VIRTIO_PCI=y
+# Without this nested-VM Kata does not work (we have not worked out exactly why)
+CONFIG_VIRTIO_PCI_LEGACY=y
+
+# CHECK - do we use the virtio HW random - I have a feeling we do...
```

Comment:
@grahamwhaley probably my actual [issue](https://github.com/kata-containers/packaging/issues/447) could help to answer this check. On s390x, the options were missing and we had too low entropy